### DISCUSSION TITLE
suggested answer is not shown

### DISCUSSION THREAD

**Discussion by ne555** [community/community#50000] [Discussions]
### Select Topic Area

Bug

### Body

in [49691](https://github.com/orgs/community/discussions/49691) it appears to have «1 suggested answer» but nothing is shown
![image](https://user-images.githubusercontent.com/1026012/224990137-64c64480-93b5-44ba-883b-489c4c06aaad.png)

**Comment by ettaboyle** [comment]
👋🏻 Thanks for flagging this @ne555 We'll let the Discussions engineering team know

**Reply by ettaboyle** [reply]
Also cheers 🥂 ... your post is the 50,000 discussion in this repo. We're grateful for you and all the other GitHub users contributing bugs, feature requests, and helping others solve their questions is this space 🚀
